basics/fish/files/bundle

```diff
@@ -0,0 +1,2 @@
+theme bobthefish
```

Comment:
We could move this into a template and make it configurable via a variable `{{ fish_theme }}`